cmd/setup/setup.go

```diff
@@ -0,0 +1,128 @@
+package setup
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	ProdJumproleConfigKey  = "prod_jumprole_account_id"
+	AwsProxy               = "aws_proxy"
+	StageJumproleConfigKey = "stage_jumprole_account_id"
+	Pd_User_Token          = "pd_user_token"
+	Jira_Token             = "jira_token"
+	Dt_Vault_Path          = "dt_vault_path"
+	Vault_Address          = "vault_address"
```

Comment:
Let's camel case these for consistency:
```suggestion
	PdUserToken            = "pd_user_token"
	JiraToken              = "jira_token"
	DtVaultPath            = "dt_vault_path"
	VaultAddress           = "vault_address"
```